src/renderer/common/sprite_batch.c

```diff
@@ -183,7 +183,7 @@ void r_flush_sprites(void) {
 }
 
 static void _r_sprite_batch_add(Sprite *spr, const SpriteParams *params, SDL_RWops *stream) {
-	SpriteAttribs alignas(32) attribs;
+	SpriteAttribs alignas(16) attribs;
```

Comment:
`SpriteAttribs` is already aligned to 16 bytes (because of the mat4 members), so there's no need to force alignment here.

Reply:
Fixed in d38a434.